plugins/in_winevtlog/in_winevtlog.c

```diff
@@ -69,7 +73,33 @@ static int in_winevtlog_init(struct flb_input_instance *in,
     }
 
     /* Set up total reading size threshold */
-    ctx->total_size_threshold = DEFAULT_THRESHOLD_SIZE;
+    if (ctx->total_size_threshold >= MINIMUM_THRESHOLD_SIZE &&
+        ctx->total_size_threshold < MAXIMUM_THRESHOLD_SIZE) {
+        flb_utils_bytes_to_human_readable_size((size_t) ctx->total_size_threshold,
+                                               human_readable_size,
+                                               sizeof(human_readable_size) - 1);
+        flb_plg_debug(ctx->ins,
+                      "read limit per cycle is set up as %s",
+                      human_readable_size);
+    }
+    else if (ctx->total_size_threshold >= MAXIMUM_THRESHOLD_SIZE) {
+        flb_utils_bytes_to_human_readable_size((size_t) MAXIMUM_THRESHOLD_SIZE,
+                                               human_readable_size,
+                                               sizeof(human_readable_size) - 1);
+        flb_plg_warn(ctx->ins,
+                     "read limit per cycle cannot exceed %s. Set up to %s",
+                     human_readable_size, human_readable_size);
+        ctx->total_size_threshold = (unsigned int) MAXIMUM_THRESHOLD_SIZE;
+    }
+    else if (ctx->total_size_threshold < MINIMUM_THRESHOLD_SIZE){
+        flb_utils_bytes_to_human_readable_size((size_t) MINIMUM_THRESHOLD_SIZE,
+                                               human_readable_size,
+                                               sizeof(human_readable_size) - 1);
+        flb_plg_error(ctx->ins,
```

Comment:
You probably meant to use `flb_plg_warn` here

Reply:
Oh, good catch. I'll fix it shortly.